content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md

```diff
@@ -21,6 +21,17 @@ weight: 10
 If you are using the Archive Service with Corda Enterprise Edition 4.11, you must use the 1.1.x stream of the Archive Service release. For more details, see [Archive Service]({{< relref "../../../../tools/archiving-service/archiving-release-notes.md" >}}).
 {{< /note >}}
 
+## Corda Enterprise Edition 4.11.4 release notes
+
+Corda Enterprise Edition 4.11.4 is a patch release of Corda Enterprise Edition focused on resolving issues.
+
+### Fixed issues
+
+* `ReleaseFinalityFlow` was returning a transaction that was missing the notary signature. This has now been fixed. The returned transaction now includes the notary signature.
+* `ReceiveTransactionFlow` was checking that the network parameters on the transaction existed before `ResolveTransactionFlow` was executed.
+This could cause a problem in certain scenarios, for example, when sending a top-level transaction to a new node in a migrated network, as the old network parameters would not exist on this new node. This has now been fixed.
```

Comment:
```suggestion
This could cause a problem in certain scenarios; for example, when sending a top-level transaction to a new node in a migrated network, as the old network parameters would not exist on this new node. This has now been fixed.
```